src/modules/delete.js

```diff
@@ -2,10 +2,10 @@ export default function deleteTodo(array) {
   array = array.filter((obj) => obj.completed === false);
 
   for (let i = 0; i < array.length; i += 1) {
```

Comment:
- [ ] OPTIONAL: It's a lot more efficient (faster) to use `map` or even `filter` instead of a regular for loop.  Try it! 💪

Reply:
Thank you! I will optimize it :)